package.json

```diff
@@ -16,63 +16,50 @@
   },
   "scripts": {
     "build": "npm-run-all --silent validate:environment list:todo-comments build:only",
-    "build:only": "rollup -c config/rollup/rollup.config.mjs",
+    "build:only": "vite build --config config/vite/vite.config.mts",
+    "build:types": "tsc",
     "delete:node_modules": "rm -rf node_modules",
     "delete:package-lock": "rm -f package-lock.json",
+    "dev": "vite dev --config config/vite/vite.config.mts",
     "format:code": "prettier --write --cache --no-editorconfig .",
-    "format:code:javascript": "prettier --write --cache --no-editorconfig \"**/*.{cjs,js,mjs}\"",
-    "format:code:json": "prettier --write --cache --no-editorconfig \"**/*.json\"",
-    "format:code:markdown": "prettier --write --cache --no-editorconfig \"**/*.md\"",
-    "format:code:yaml": "prettier --write --cache --no-editorconfig \"**/*.yml\"",
-    "lint:javascript": "echo Linting JavaScript... && eslint --ext cjs,js,mjs --max-warnings 0 --cache --cache-location .eslintcache --cache-strategy content .",
+    "lint:typescript": "echo Linting TypeScript... && eslint --ext cts,ts,mts --max-warnings 0 --cache --cache-location .eslintcache --cache-strategy content .",
     "list:eslint:disable-directives": "node_modules/@vscode/ripgrep/bin/rg '/(/|\\*+)[ \\t]*eslint-disable[^*]*(\\*+/)?'",
     "list:todo-comments": "node_modules/@vscode/ripgrep/bin/rg --only-matching '(TODO|FIXME):[a-zA-Z0-9\\t .,;?!@#$%^&*()]+'",
     "reinstall": "npm run --silent delete:node_modules && npm run --silent delete:package-lock && npm i",
     "reinstall:use-lock-file": "npm run --silent delete:node_modules && npm ci",
-    "test": "npm run --silent validate:environment && jest --config config/jest/jest.config.mjs --coverage --rootDir .",
-    "test:only": "jest --config config/jest/jest.config.mjs --coverage --rootDir .",
-    "test:watch": "npm run test -- --watch",
-    "update:package-versions": "./scripts/npm/updatePackageVersions.mjs --ignore-packages=rollup && npm run --silent reinstall",
+    "test": "npm-run-all --silent validate:environment test:unit",
+    "test:unit": "vitest --config config/vite/vite.config.mts --coverage",
+    "test:unit:ui": "npm run --silent test:unit -- --ui",
```

Comment:
This is a really nice feature, worth trying when you have time. It provides a nice UI for feedback for your running tests.